src/into.rs

```diff
@@ -41,7 +41,7 @@ pub fn into_impl(ast: DeriveInput) -> TokenStream {
         impl #impl_generics ::pyo3::IntoPy<::pyo3::PyObject> for #name #ty_generics #where_clause {
             fn into_py(self, py: ::pyo3::Python) -> ::pyo3::PyObject {
                 use ::pyo3::{IntoPy, PyObject, PyErr, PyResult};
-                use ::pyo3::exceptions::{ValueError, TypeError};
+                use ::pyo3::exceptions::{PyValueError, PyTypeError};
```

Comment:
Now that I think about it, it looks like this import is dead code, otherwise changing it should have caused other errors. Can you confirm @gperinazzo?

Reply:
Yes, it seems those are no longer used. That and PyResult.

Reply:
Cool, I'll remove them then!